packages/tiled/lib/src/tiled_map.dart

```diff
@@ -279,6 +282,24 @@ class TiledMap {
     throw ArgumentError('Layer $name not found');
   }
 
+  /// Finds the [TiledObject] in this map with the unique [id].
```

Comment:
Might as well mention that the first call will cache all the objects. Based on how complex the map is, users might want to avoid making the first call during the gameplay. It is highly unlikely, but we never know 😄

Reply:
👍 good idea, done